### ISSUE TITLE
Where/How to Access Yahoo Finance's Qtrly Analyst Estimates for Revenues & EPS using yahoofinance? like  .get_estimates_analysis() or something?

### ISSUE THREAD

**Issue by SymbReprUnlim** [JECSand/yahoofinancials#161]
Yahoo Finance has a page for stock tickers that includes Quarterly Analyst Estimates (Avg, High, Low) for Revenues and EPS, e.g. for NVDA the web page address is  https://finance.yahoo.com/quote/nvda/analysis/

NVDA on Yahoo Finance has info like:

Earnings Estimate | Current Qtr. (Oct 2023) | Next Qtr. (Jan 2024) | Current Year (2024) | Next Year (2025)
-- | -- | -- | -- | --
No. of Analysts | 33 | 32 | 40 | 40
Avg. Estimate | 3.36 | 3.74 | 10.86 | 17.14
Low Estimate | 3.25 | 3.33 | 10.1 | 12.31
High Estimate | 3.57 | 4.5 | 11.87 | 22.77
Year Ago EPS | 0.58 | 0.88 | 3.34 | 10.86

Revenue Estimate | Current Qtr. (Oct 2023) | Next Qtr. (Jan 2024) | Current Year (2024) | Next Year (2025)
-- | -- | -- | -- | --
No. of Analysts | 32 | 31 | 44 | 45
Avg. Estimate | 16.12B | 17.74B | 54.92B | 82.87B
Low Estimate | 16B | 16.5B | 53.2B | 68.61B
High Estimate | 17.02B | 20.8B | 58.51B | 113.36B
Year Ago Sales | N/A | 6.05B | 26.97B | 54.92B
Sales Growth (year/est) | N/A | 193.10% | 103.60% | 50.90%

Earnings History | 10/30/2022 | 1/30/2023 | 4/29/2023 | 7/30/2023
-- | -- | -- | -- | --
EPS Est. | 0.69 | 0.81 | 0.83 | 2.09
EPS Actual | 0.58 | 0.88 | 0.98 | 2.7
Difference | -0.11 | 0.07 | 0.15 | 0.61
Surprise % | -15.90% | 8.60% | 18.10% | 29.20%

EPS Trend | Current Qtr. (Oct 2023) | Next Qtr. (Jan 2024) | Current Year (2024) | Next Year (2025)
-- | -- | -- | -- | --
Current Estimate | 3.36 | 3.74 | 10.86 | 17.14
7 Days Ago | 3.36 | 3.73 | 10.85 | 17.11
30 Days Ago | 3.35 | 3.7 | 10.79 | 17.06
60 Days Ago | 3.34 | 3.67 | 10.76 | 16.71
90 Days Ago | 2.35 | 2.56 | 8.07 | 11.56

EPS Revisions | Current Qtr. (Oct 2023) | Next Qtr. (Jan 2024) | Current Year (2024) | Next Year (2025)
-- | -- | -- | -- | --
Up Last 7 Days | 1 | 1 | 1 | 1
Up Last 30 Days | 3 | 4 | 5 | 3
Down Last 7 Days | N/A | N/A | N/A | N/A
Down Last 30 Days | N/A | N/A | N/A | N/A

Growth Estimates | NVDA | Industry | Sector(s) | S&P 500
-- | -- | -- | -- | --
Current Qtr. | 479.30% | N/A | N/A | N/A
Next Qtr. | 325.00% | N/A | N/A | N/A
Current Year | 225.10% | N/A | N/A | N/A
Next Year | 57.80% | N/A | N/A | N/A
Next 5 Years (per annum) | 78.70% | N/A | N/A | N/A
Past 5 Years (per annum) | 37.03% | N/A | N/A | N/A

Then, also on the same webpage, but on the right Yahoo Finance has tables like:

Similar to NVDA:
Symbol | Last Price | Change | % Change
-- | -- | -- | --
AMD  Advanced Micro Devices, Inc. | 118.00 | -1.88 | -1.57%
TSM  Taiwan Semiconductor Manufacturing Company Limited | 98.80 | -0.11 | -0.11%
ARM  Arm Holdings plc | 55.43 | +2.00 | +3.74%
INTC  Intel Corporation | 40.61 | +1.20 | +3.04%
AVGO  Broadcom Inc. | 975.40 | +2.44 | +0.25%

[Recommendation Rating] (https://finance.yahoo.com/quote/NVDA/analysis?p=NVDA)
1.7
Where
    1 =     Strong     Buy
    2 =     Buy
    3 =     Hold
    4 =     Under-perform
    5  =    Sell
Current price $488.88

In Short, How to access any/all of the information above using yahoofinance with Python, ref. https://pypi.org/project/yahoofinancials/

Thank You!

**Comment by JECSand** [comment]
@SymbReprUnlim I'm looking into this.

**Comment by elmel01** [comment]
Hi Connor, can we also retrieve the sector average for each metric from yahoo finance? Or at least any idea of how we can calculate. I am developing a code that shows the performance of the company by comparing its metrics with the sector average. I really appreciate any help you can provide. :)
